### PR TITLE
Update contributor info

### PR DESCRIPTION
**Pull request by geealbers**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

no

### Please describe the goal of this pull request and the changes that were made.

It started because the page YAML value `contributor_byline` needed to be changed to `byline_format`, but I took some extra time to look at the page overall to update it for correctness and clarity.

### Additional Comments

I wasn't sure how we were handling figures that needed to be different from the v0 to the v1 docs. In this case, the caption for the image was correct for v0 but needed to be different for v1. So, I created a new figures.yaml entry using the same `src` image but with the updated caption, so that the original image and caption would still persist in the v0 docs.

### CODE REVIEW

File: content/docs-v1/contributors.md [hunk 1]
```diff
@@ -63,44 +63,50 @@ contributor:
...
 ```
-While `name-title` is the default (and will be used if no value is specified), `name` will omit any title or affiliation information, and `false` will remove the contributor listing from the heading of the page altogether.
 
-{{< q-figure id="modern_contributors-page" >}}
+{{< q-figure id="modern_contributors-page-v1" >}}
 
 {{< q-class "box tip" >}}
 
-- If you specify `bylineFormat: false` in the `config.yaml` you can still have names appear on individual pages by specifying either `contributor_byline: name-title` or `contributor_byline: name` on that page.
+- If you specify `bylineFormat: false` in the `config.yaml` you can still have names appear on individual pages by specifying either `byline_format: name-title` or `byline_format: name` on that page.
```
Comment (Erin-Cecele) [inline content/docs-v1/contributors.md:99]: Does "on that page" mean in the page YAML?

**Review by Erin-Cecele — APPROVED** [review]
Just one small question, otherwise looks great.

File: content/docs-v1/contributors.md (hunk 1 above)
Comment (geealbers) [reply content/docs-v1/contributors.md:99]: Good question, and the answer is yes. This is clearer I think?:

> ... on an individual page by specifying either `byline_format: name-title` or `byline_format: name` in the YAML for that page.

Comment (Erin-Cecele) [reply content/docs-v1/contributors.md:99]: Yes! Looks good.

**Pull request merged**